mobile/src/main/java/org/openhab/habdroid/model/Item.kt

```diff
@@ -222,15 +354,149 @@ fun String?.toItemType(): Item.Type {
 
 fun String?.toItemTag(): Item.Tag {
     return when (this) {
-        "Lighting" -> Item.Tag.Lighting
-        "Switchable" -> Item.Tag.Switchable
+        "Lighting" -> Item.Tag.Light
+        "Switchable" -> Item.Tag.Switch
         "ContactSensor" -> Item.Tag.ContactSensor
-        "CurrentTemperature" -> Item.Tag.CurrentTemperature
-        "CurrentHumidity" -> Item.Tag.CurrentHumidity
-        "Thermostat" -> Item.Tag.Thermostat
+        "CurrentTemperature" -> Item.Tag.Temperature
+        "CurrentHumidity" -> Item.Tag.Humidity
+        "Thermostat" -> Item.Tag.Temperature
         "homekit:HeatingCoolingMode", "homekit:TargetHeatingCoolingMode" -> Item.Tag.HeatingCoolingMode
         "homekit:TargetTemperature", "TargetTemperature" -> Item.Tag.TargetTemperature
-        "WindowCovering" -> Item.Tag.Blind
+        "WindowCovering" -> Item.Tag.Blinds
+
+        "Equipment" -> Item.Tag.Equipment
```

Comment:
I'd suggest handling the 1:1 mapping in a default case, using `Item.Tag.valueOf()`. That way one can get rid of the boilerplate code.